pheweb/serve/data_access/pqtl_colocalization.py

```diff
@@ -69,4 +74,40 @@ def get_pqtl_colocalization(self, gene_name: str):
                 result.append(pqtl)
                 
             return result
+        
+
+    def get_gene_colocalization(self, gene_name: str):
+
+        with closing(self.get_connection()) as conn:
+            fields = self._fields
+            tables = [field['table'] for field in fields]
+            coloc_table_id = tables.index([t for t in tables if 'coloc' in t][0])
```

Comment:
It would be very much preferable to expect tables with certain exact identifiers than rely on prefix.   give them names e.g. 
{ 'name':'colocalization', 'table':'whateveritis', .... } 
then check that all required ones are defined and error out if not

Reply:
Changed to 
```
{ "pqtl_colocalization": 
      { "PqtlColocalisationDao": 
        { "authentication_file": "...",
          "pqtl": {
            "table": "pqtl_finemap",
            "columns": [...] 
          },
          "colocalization": {
              "table": "colocalization",
              "columns": [...]
          } 
      } } 
    }
```

would this work?

Reply:
yes that looks great!!!